CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,88 @@
+
+Media shared through public infrastructure run by the IPFS team must not contain illegal or infringing content. You should only publish content via IPFS public infrastructure if you have the right to do so. This includes complying with all software license agreements or other intellectual property restrictions. You will be solely responsible for any violation of laws or
+others’ intellectual property rights.
+
+## Reporting Violations of this Code of Conduct
+
+If you believe someone is harassing you or has otherwise violated this Code of Conduct, please [contact us](#contact-info) to send us an abuse report. If this is the initial report of a problem, please include as much detail as possible. It is easiest for us to address issues when we have more context.
+
+## Consequences
+
+Unacceptable behavior from any community member will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the team may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event or service).
+
+## Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify the team. We will do our best to ensure that your grievance is handled appropriately.
+
+In general, we will choose the course of action that we judge as being most in the interest of fostering a safe and friendly community.
+
+On IRC, let one of the ops know if you think that someone has transgressed against the Code of Conduct. If you would like to be an op and promise to help maintain and abide by the code, please let us know.
+
+## Contact Info
+
+The Drand team currently defers to the IPFS abuse team for Code of Conduct resolutions. Please contact abuse@ipfs.io if you need to report a problem or address a grievance related to an abuse report.
+
+We will keep all matters confidential and they will only be shared within the team, with the exception of legal counsel when deemed necessary, or unless given explicit permission by the reporter.
+
+You are also encouraged to contact us if you are curious about something that might be "on the line" between appropriate and inappropriate content. We are happy to provide guidance to help you be a successful part of our community.
+
+## Changes
+
+This is a living document and may be updated from time to time. Please refer to the git history for this document to view the changes.
+
+## Credit and License
```

Comment:
This feels like a better place to say who will be dealing with the help requests (IPFS abuse).